aten/src/ATen/native/cuda/LinearAlgebra.cu

```diff
@@ -44,14 +44,16 @@ Tensor& addbmm_cuda_out(Tensor& result, const Tensor& self, const Tensor& batch1
   return legacy::cuda::_th_addbmm_out(result, self, batch1, batch2, beta, alpha);
 }
 
-Tensor bmm_cuda(const Tensor& self, const Tensor& mat2) {
-  return legacy::cuda::_th_bmm(self, mat2);
```

Comment:
curious, does this removal mean there's now a generated `legacy::cuda::_th_bmm` that nobody calls?

Reply:
yes, there's no way to control that separately.  I've considered adding another annotation in order to reduce the amount of generated code, but haven't done it yet.